docs/swagger.json

```diff
@@ -965,43 +984,41 @@
         "responses": {}
       }
     },
-    "/pinning/pins/{id}": {
-      "post": {
-        "description": "This endpoint replaces a pinned object.",
+    "/pinning/pins/{requestid}": {
+      "get": {
+        "description": "This endpoint returns a pinned object.",
         "produces": [
           "application/json"
         ],
         "tags": [
           "pinning"
         ],
-        "summary": "Replace a pinned object",
+        "summary": "Get a pinned objects",
         "parameters": [
           {
             "type": "string",
-            "description": "id",
-            "name": "id",
+            "description": "cid",
```

Comment:
This is not the CID, it's the pin ID (which is given when the pin request is made). Maybe `pinid` instead?